edb/schema/constraints.py

```diff
@@ -534,6 +571,7 @@ def get_ref_implicit_base_delta(
             if not b.get_abstract(schema)
             and b.generic(schema) and b.get_name(schema) != default_base
         ]
+        # assert not explicit_bases
```

Comment:
Uncomment or delete?

Reply:
Let's try uncommenting it, and if that works I'll delete the apparently dead code above it.

(The explicit_bases condition requires `get_abstract` and `generic` to disagree and I don't understand when that would happen?)

Reply:
:heavy_check_mark: